components/server/src/ome/security/SystemTypes.java

```diff
@@ -78,6 +79,8 @@ public boolean isSystemType(Class<?> klass) {
             return true;
         } else if (IGlobal.class.isAssignableFrom(klass)) {
             return true;
+        } else if (Namespace.class.isAssignableFrom(klass)) {
```

Comment:
The codegen still seems to generate some kind of annotation count table for the namespace but without then replacing it with a view. I'm guessing that globals aren't expected to be annotatable.